ChangeBGColor/index.js

```diff
@@ -0,0 +1 @@
+document.querySelector("input").addEventListener("change", () => document.body.style.background = button.value);
```

Comment:
O `button` não está atribuido 
```suggestion
const button = document.querySelector("input");
button.addEventListener("change", () => document.body.style.background = button.value);
```